subsys/net/ip/l2/ieee802154/Kconfig

```diff
@@ -18,6 +18,10 @@ config NET_L2_IEEE802154_MGMT
 	select NET_MGMT_EVENT
 	default n
 
+config NET_L2_IEEE802154_SUB_GHZ
+	bool
+	default n
```

Comment:
This needs to have a "help" block to give the future user some guidance.

Reply:
done